src/core/misc.c

```diff
@@ -218,6 +218,19 @@ GSList *gslist_remove_string (GSList *list, const char *str)
 	return list;
 }
 
+GSList *gslist_delete_string (GSList *list, const char *str, GDestroyNotify free_func)
```

Comment:
you can remove the remove_string method, which is apparently flawed

Reply:
lol this comment came back